syntax = "proto3";
message Agent_Info{
	fixed32 Agent_Action = 1;
	fixed32 Current_Position = 2;
	string Agent_Trading_Instrument = 3;
	string Action_Timestamp = 4;
	bool LatestResult = 5;
	bool msg_pipe_init = 6;
};

message MD_Info{
	double last_price = 1;
	fixed32 volume = 2;
	bool msg_pipe_init= 3;
	string MD_Timestamp = 4;
	string MD_Instrument = 5;
};
